addons/main/CfgVehicles.hpp

```diff
@@ -24,14 +24,14 @@ class CfgVehicles
 		icon = "iconObject_1x1";
 	};
 
-     class Man;
-     class CAManBase: Man
-     {
-          class ACE_SelfActions
-          {
-               class ACE_Equipment {
-                    #include "ACE_SelfActions.hpp"
-               };
-          };
-     };
+    class Man;
+    class CAManBase: Man
+    {
+        class ACE_SelfActions
+        {
+			class ACE_Equipment {
+                #include "ACE_SelfActions.hpp"
+			};
+		};
+	};
 };
```

Comment:
What is this formatting? :D 
```suggestion
        class ACE_SelfActions
        {
            class ACE_Equipment {
                #include "ACE_SelfActions.hpp"
            };
        };
    };
};
```

Reply:
/shrug

Spacing was all off and my editor was angry, so I fixed it